src/Policy/AndPolicy.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Chubbyphp\Serialization\Policy;
+
+use Chubbyphp\Serialization\Normalizer\NormalizerContextInterface;
+
+final class AndPolicy implements PolicyInterface
+{
+    /**
+     * @var array|PolicyInterface[]
+     */
+    private $policies;
+
+    /**
+     * @param array|PolicyInterface[] $policies
+     */
+    public function __construct(array $policies)
+    {
+        $this->policies = $policies;
+    }
+
+    /**
+     * @param NormalizerContextInterface $context
+     * @param object|mixed               $object
+     *
+     * @return bool
+     */
+    public function isCompliant(NormalizerContextInterface $context, $object): bool
+    {
+        foreach ($this->policies as $policy) {
+            if (false === $policy->isCompliant($context, $object)) {
```

Comment:
Sometimes you use `!` and sometimes `false !==` maybe standardize the way for comparisons.

Reply:
done